File: Ex1.py
x=4

stars_n=1

for i in range(1,x+1):
    spacing_n=int((x-i))
    spacing=" "*spacing_n
    output="*"*stars_n
    print(spacing+output)
    stars_n+=2

starts_n=x
for i in range(x,1,-1):
    spacing_n=int((x-i))
    spacing=" "*spacing_n
    
    output="*"*stars_n
    print(spacing+output)
    stars_n-=2
    

File: Ex1/Ex1.js
var rows = 5

var stars_n = 1

for(let i=1;i<=rows;i++){
    var spacing_n=rows-i
    var spacing = " ".repeat(spacing_n)

    var output = "*".repeat(stars_n)
    console.log(spacing+output)

    if(i<rows)
        stars_n=stars_n+2
}

for(let i=rows;i>0;i--){

    stars_n-=2

    var spacing_n=rows-i+1
    var spacing = " ".repeat(spacing_n)

    var output = "*".repeat(stars_n)
    console.log(spacing+output)

}
